zcash_client_backend/src/wallet.rs

```diff
@@ -98,6 +98,7 @@ impl<AccountId, N> Recipient<AccountId, Option<N>> {
 /// The shielded subset of a [`Transaction`]'s data that is relevant to a particular wallet.
 ///
 /// [`Transaction`]: zcash_primitives::transaction::Transaction
+#[derive(Clone)]
```

Comment:
I had to do this  for my version of `put_blocks` as i was not able to make it otherwise, it might be not needed and not wanted.